packages/ember-metal/lib/property_events.js

```diff
@@ -108,6 +111,13 @@ function propertyDidChange(obj, keyName) {
   }
 
   markObjectAsDirty(m);
+
+  if (isEnabled('ember-glimmer-detect-two-way-reflush') ||
+      isEnabled('ember-glimmer-allow-two-way-reflush')) {
+    if (has('glimmer-runtime')) {
```

Comment:
If we made them no-ops, then we can also remove the stub of `runInTransacation` in the renderer